pulumi/grapl/__main__.py

```diff
@@ -93,6 +93,15 @@ def _container_images(artifacts: ArtifactGetter) -> Mapping[str, DockerImageId]:
     }
 
 
+def _container_versions(
+    container_images: Mapping[str, DockerImageId]
+) -> Mapping[str, DockerImageVersion]:
+    container_versions = dict()
+    for key, value in container_images.items():
+        container_versions[key] = value.split(":")[1]
+    return container_versions
```

Comment:
It looks like this logic is the same in all the Pulumi projects; can we create a common implementation to cut down the repetition?